Kickstarter-iOS/Views/Controllers/BackerDashboardProjectsViewController.swift

```diff
@@ -0,0 +1,118 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+internal protocol BackerDashboardProjectsViewControllerDelegate: class {
+  /// Called when a project cell is tapped.
+  func profileProjectsGoToProject(_ project: Project, projects: [Project], reftag: RefTag)
```

Comment:
curious why you need to tell the delegate to go to the project. could you not present the project straight from this view controller?